stdlib/public/runtime/StaticBinaryELF.cpp

```diff
@@ -61,6 +64,35 @@ typedef Elf32_Section Elf_Section;
 
 extern const Elf_Ehdr elfHeader asm("__ehdr_start");
 
+static unsigned long getAuxVal(unsigned long type) {
```

Comment:
Why not provide the function only in the case that the underlying system does not provide it and name it identically?  It has internal linkage, so it should remain forward compatible.

Reply:
Agree, done